manuscript/the-meat.txt

```diff
@@ -188,20 +188,20 @@ You should now see a histogram like this:
 
 ## Adding an axis
 
-Axes are one of my favourite feats of d3.js magic. You just call a function, set some parameters and bam, you’ve got an axis.
+Axes are one of my favourite feats of d3.js magic. You simply call a function, set some parameters, and **BAM**, you’ve got an axis.
```

Comment:
Let's drop the `simply` it doesn't add anything.

In general I think most `just`s and `simply`s and things like that can be removed. Feel free to call me out on my shit and tell me when I'm using words that don't add anything to the sentence.